src/component/axis/AngleAxisView.ts

```diff
@@ -138,18 +138,24 @@ const angelAxisElementsBuilders: Record<typeof elementList[number], AngleAxisEle
 
     axisLine(group, angleAxisModel, polar, ticksAngles, minorTickAngles, radiusExtent) {
         const lineStyleModel = angleAxisModel.getModel(['axisLine', 'lineStyle']);
+        const angleAxis = polar.getAngleAxis();
+        const RADIAN = Math.PI / 180;
+        const angleExtent = angleAxis.getExtent();
 
         // extent id of the axis radius (r0 and r)
         const rId = getRadiusIdx(polar);
         const r0Id = rId ? 0 : 1;
-
         let shape;
+
         if (radiusExtent[r0Id] === 0) {
-            shape = new graphic.Circle({
+            shape = new graphic.Arc({
```

Comment:
I run the visual tests (`npm run test:visual`) and find there are some subtle diffs with `polarLine.html`, which is a case when `startAngle` is not the default value. I would suggest using `graphic.Circle` when `Math.abs(angleExtent[1] - angleExtent[0]) === 360` (or is there any other situations?) and use `graphic.Arc` otherwise.